### DISCUSSION TITLE
Addon arm version return?

### DISCUSSION THREAD

**Discussion by shirou93** [hassio-addons/addon-traccar#159] [Q&A] [answered]
I want to buy RPI4 and this addon not support arm?

I remember when arm64 have support. Why actual version don't have have arm CPU support?

Official docker traccar have arm64 version. https://github.com/traccar/traccar-docker

**Comment by frenck** [comment, marked as answer]
This addon has arm 64 bits support?

**Reply by shirou93** [reply]
Ok thanks. I did not understand that aarch64 is arm64.
